packages/nimble-components/src/rich-text-mention/users/view/styles.ts

```diff
@@ -32,4 +33,8 @@ export const styles = css`
     :host([disable-editing]) slot {
         display: none;
     }
+
+    :host([disable-editing]) {
+        font: ${mentionFont};
+    }
```

Comment:
This should be next to the other host selectors to be in DOM order, see: https://github.com/ni/nimble/blob/main/packages/nimble-components/docs/css-guidelines.md#group-selectors-by-target-and-document-order

Reply:
Updated.